tests/app/questionnaire/routing/test_operators.py

```diff
@@ -76,6 +79,24 @@ def test_operation_greater_than(operands, expected_result):
     assert operator.evaluate(operands) is expected_result
 
 
+@pytest.mark.parametrize(
+    "operands, expected_result",
+    greater_than_and_less_than_operations_equals_test,
+)
+def test_operation_greater_than(operands, expected_result):
```

Comment:
There are three tests here with the same name, it's a bit confusing on what's being tested